src/main/java/org/cs23sw612/Util/AlphabetUtil.java

```diff
@@ -33,6 +33,8 @@ private static Boolean[] booleanArrayFromInt(int x, int size) {
     public static String toBinaryString(Object obj) {
         if (obj instanceof Boolean)
             return (Boolean) obj ? "1" : "0";
+        else if (obj instanceof Bit)
+            return ((Bit) obj).value ? "1" : "0";
         else if (obj instanceof Number)
             return ((Number) obj).longValue() != 0 ? "1" : "0";
         else if (obj instanceof Word<?>)
```

Comment:
kunne være nice at bruge overloads eller interface i stedet for :D

Reply:
TIl det fjerde case (`Word<?>`) vil overload ik virke fordi typer
Men funktionen forsvinder med den nye impl, vil forslå at lade den være indtil da..??

Reply:
Yes det er fint, java er troll